advanced_functionality/multi-container-endpoint/direct-invocation/multi-container-direct-invocation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>A few grammar nits:</p><p> </p><ul><li>Missing space for <code>algorithm(not</code></li><li>No need to capitalize D in <code>For Deep learning</code></li><li>Best practice to make the link text the name of the page, so you could change the text to <code>refer to Available Deep Learning Containers Images</code> with the same hyperlink rather than <code>refer to this GitHub Repository</code>  </li><li>Missing period at end of paragraph</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>